lunes_cms/static/js/toggle_plural_field.js

```diff
@@ -0,0 +1,8 @@
+toggleFields($('#id_word_type').val())
+$('#id_word_type').change(function(){
+    toggleFields($(this).val())
+})
+
+function toggleFields(wordType) {
+    $("#id_plural").closest("div.row").toggle(wordType == "Nomen");
+}
```

Comment:
[This SO answer](https://stackoverflow.com/a/16014827) seemed to help:

```suggestion
if (!$) {
  $ = django.jQuery;
}

$(document).ready(() => {
  $("#id_word_type").change((event) =>
    $("#id_plural")
      .closest("div.row")
      .toggle($(event.target).val() == "Nomen")
  );
  $("#id_word_type").trigger("change");
});

```

Reply:
Amazing! Thank you!